mono/profiler/proflog.h

```diff
@@ -93,6 +93,7 @@ enum {
 	TYPE_END
 };
 
+// Sampling modes
```

Comment:
This comment can be a bit confusing since we have another enum in `metadata/profiler.h` that is called `MonoProfileSamplingMode`.